packages/eas-cli/src/graphql/queries/credentials/AppleTeamQuery.ts

```diff
@@ -0,0 +1,34 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../../client';
+import { AppleTeam } from '../../types/credentials/AppleTeam';
+
+export class AppleTeamQuery {
+  static async byAppleTeamIdentifierAsync(
```

Comment:
```suggestion
const AppleTeamQuery = {
  byAppleTeamIdentifierAsync(
```
Maybe? This class is never initialized so there's no need for it to be a class.